4.component.md

```diff
@@ -1,5 +1,5 @@
 # 4. The Component Model
-**Target Audience: Application Developer** 
+**Target Audience: Application Developer, Application Operator** 
```

Comment:
What does an application operator do with a component?

Reply:
Mainly to know what parameters they can passthrough.